deploy.go

```diff
@@ -93,6 +93,12 @@ func NewGateway(host string, username, password string, insecure, useCerts bool)
 		}
 	}
 
+	// For versions greater than 3.5 we need the token in order to get the version.
+	token, errT := gc.NewTokenGeneration()
```

Comment:
Any reason it's `errT` instead of `err`?

Reply:
nope, i'll change it to err